docs/configs/override-pages.md

```diff
@@ -0,0 +1,365 @@
+# Override Pages
+In some cases, the default pages do not offer all the functionality or modification possibilities needed. It is possible to override the pages, though this should be the last resort as it increases resources needed for maintainability.
+
+! IMPORTANT: This is valid for `invenio-app-rdm==v6.0.2` !
+
+
+### Troubleshoot
+
+#### ManifestKeyNotFoundError
+- check if the spelling of the webpack names match with the import in the template.
+- reinstall invenio-theme-tugraz
+    - In order to pick up all the new files and to rebuild webpack, it is necessary to install `invenio-theme-tugraz` again. If no such changes happened, this step can be skipped.
+    - `invenio-cli packages install /path/to/invenio-theme-tugraz`
+
+
+### JavaScript and React Components
+In order to provide functionality, all the needed components either have to be created or imported from other modules. To achieve this, create or update the following file `invenio_theme_tugraz/assets/semantic-ui/js/invenio_theme_tugraz/{componentToOverride}/components.js`. This will make it easier to keep a good overview of the dependencies.
+
+Add all needed imports:
+```js
+import { Button, Card} from "semantic-ui-react";
+
+// import component from invenio module
+import { SearchBar } from "@js/invenio_search_ui/components";
+```
+
+Create new components:
+```js
+export const RDMRecordSearchBarContainer = () => {
+  return (
+    <Overridable id={"SearchApp.searchbar"}>
+      <SearchBar />
+    </Overridable>
+  );
+};
```

Comment:
this does not make sense - if this belongs to the override of the search page - add it to the section.